tools/setup.py

```diff
@@ -56,33 +56,38 @@ def prepare_executable(original_nso: Optional[Path]):
     if not TARGET_ELF_PATH.is_file():
         setup.fail("internal error while preparing executable (missing ELF); please report")
 
-def create_build_dir(ver):
+def get_build_dir():
+    return setup.ROOT / "build"
+
+def create_build_dir(ver, cmake_backend):
     if(ver != Version.VER_100): return # TODO remove this when multiple versions should be built
-    build_dir = setup.ROOT / "build" # ("build-"+ver.value)
+    build_dir = get_build_dir() # ("build-"+ver.value)
```

Comment:
Move the comment up with the other code

Reply:
that comment doesn't even make sense, so i'll remove it

Reply:
It was some code from when I experimented with multi-versioned stuff, but ... most of it is removed either way, and archived back in a different branch, so feel free to remove it as well.